### PR TITLE
test: extend test runner to print errors (fixes #51)

### LINKED ISSUE #51: extend test runner to print errors
With the upcoming refactoring of the Kataw parser, the error handling is an callback func. So it's needed to extend the
test runner to print in each '.md' errors if they are returned by the cb.

As of now we will never know if any error diagnostics has been thrown

### PR DESCRIPTION
**Pull request by aladdin-add**
test needs to be updated.

### CODE REVIEW

File: test/runner/tob.ts [hunk 1]
```diff
@@ -137,7 +135,7 @@ function diagnostics2md(diagnostics: any) {
   let diagnosticString = '';
   if (diagnostics.length) {
     diagnostics.forEach(function (a: any) {
-      diagnosticString += '✖ ' + a.message + ' - start: ' + a.start + ', end: ' + a.length;
+      diagnosticString += '✖ ' + a[1] + ' - start: ' + a[2] + ', end: ' + a[3];
```
Comment (aladdin-add) [inline test/runner/tob.ts:138]: * a[0] => code
* a[1] => message
* a[2] => start
* a[3] => end

was it right?

**Review by KFlash — APPROVED** [review]

File: test/runner/tob.ts (hunk 1 above)
Comment (KFlash) [reply test/runner/tob.ts:138]: yes. The code is either lexer or parser. Not important here. 2nd arg is message. then start and end

Comment (KFlash) [reply test/runner/tob.ts:138]: .length is wrong. should be 'a .end'. same as you see on the CST node

**Comment by aladdin-add** [comment]
I have added the commit to `cst-parser` branch, so this can be closed now. :)

**Pull request closed without merging**